main.go

```diff
@@ -214,6 +214,8 @@ func buildCredentials(skipVerify bool, caCerts, clientCert, clientKey, serverNam
 	return credentials.NewTLS(&cfg), nil
 }
 
+var versionTag = "" // set from git tag via ldflags during build
```

Comment:
We can also default to eg. `"development"`:
```suggestion
var versionTag = "development" // set from git tag via ldflags during build
```
and remove the condition lower to keep code simpler and provide consistent output structure.

Reply:
Yeah sounds good. I initially deemed tag is not so useful and anyone who needs can derive it from commit ID.

Reply:
"Sounds good" as in omiting the tag if not set via `ldflags` or showing the "development" in such cases?